src/Workspaces/CSharp/Portable/Editing/CSharpImportAdder.cs

```diff
@@ -194,6 +194,9 @@ private void CheckName(NameSyntax node)
                 if (node.IsRightSideOfDotOrArrowOrColonColon())
                     return;
 
+                if (node.IsVar)
```

Comment:
I see, so from my side I don't see any harm for this check

Reply:
i would def doc this.